user-interface/src/main/java/life/qbic/datamanager/views/projects/create/AddProjectDialog.java

```diff
@@ -153,23 +157,22 @@ private void onCancelClicked() {
   }
 
   private void onConfirmClicked(ClickEvent<Button> event) {
-    if (projectDesignLayout.validate().isInvalid()) {
+    if(projectDesignLayout.validate().isInvalid()){
       return;
     }
-    if (fundingInformationLayout.validate().isInvalid()) {
+    projectCreationInformation.setProjectDesign(projectDesignLayout.getProjectDesign());
```

Comment:
Why did you decide to go with a mutable object that is filled step by step instead of handing off the information once at the end? What are the advantages?

Reply:
Valid Point, i thought it's easier to understand since the object is filled step by step but it doesn't matter in the end